Testing/xASL_test_IntegrationTesting.m

```diff
@@ -232,12 +232,19 @@
 %% Write logging information to XLSX table
 function xASL_test_writeLoggingTable(filePath,logTable)
 
-    % Write file
+    % Write XLSX file
     try
-        writetable(logTable,filePath,'WriteVariableNames',true);
+        writetable(logTable,[filePath '.xlsx'],'WriteVariableNames',true);
```

Comment:
Why is `xlsx` so useful, can we not keep TSV only? That would be better for multiple reasons: it is more open-access, easier to read, lighter weight.

Reply:
Agree. TSV is easy to import to Excel. Was there some special reason @MichaelStritt ?

Reply:
I don't know... I just like it because I can double click open it with excel or libre office and it's nicer too look at than a tsv text editor. This isn't part of the pipeline where we have to be 100% bids compliant. We had the exact same discussion with the previous versions a few months ago as well though, I think we just have a different opinion here.

Reply:
ah - true. That's a test only. Fine to keep XLSX. Actually - my editor opens Excel with tsvs as well - that's why I don't care that much :D